app/components/post_draft/quick_actions/camera_quick_action/index.js

```diff
@@ -147,17 +148,17 @@ export default class CameraQuickAction extends PureComponent {
     };
 
     render() {
-        const {disabled, theme} = this.props;
-        const testID = disabled ?
-            'post_draft.camera_quick_action.disabled' :
-            'post_draft.camera_quick_action';
+        const {testID, disabled, theme} = this.props;
+        const actionTestID = disabled ?
+            `${testID}.disabled` :
```

Comment:
Here is an example of string interpolation. Can we do this in other places? Keep it consistent and also string concatenation make me sad 😢

Reply:
Updated